p2p/config.go

```diff
@@ -48,6 +51,8 @@ type Config struct {
 	RandSeed int64
 	// Disable bootstrapping altogether. BootstrapNodes has no effect over this.
 	NoBootstrap bool
+	// Disables gossiping
+	NoGossip bool
```

Comment:
This is a solution for testing that also seemed useful as a configuration option.